Tests/Test/ResponseTest.swift

```diff
@@ -101,11 +115,44 @@ class ResponseTest: XCTestCase {
     }
 
     func testBuildResponseWillDecodeOperators() {
-        let request = "GET /parameters?variable_1=Operators%20%3C%2C%20%3E%2C%20%3D%2C%20!%3D%3B%20%2B%2C%20-%2C%20*%2C%20%26%2C%20%40%2C%20%23%2C%20%24%2C%20%5B%2C%20%5D%3A%20%22is%20that%20all%22%3F"
+        let request = "GET /parameters?variable_1=Operators%20%3C%2C%20%3E%2C%20%3D%2C%20!%3D%3B%20%2B%2C%20-%2C%20*%2C%20%26%2C%20%40%2C%20%23%2C%20%24%2C%20%5B%2C%20%5D%3A%20%22is%20that%20all%22%3F\r\nConnection: Keep-Alive\r\n\r\n"
+
         let expectedResponse = "HTTP/1.1 200 OK\r\nContent-Length: 127\r\nContent-type: text/html\r\n\r\n<!DOCTYPE html><html><body><h1>variable_1 = Operators <, >, =, !=; +, -, *, &, @, #, $, [, ]: \"is that all\"?</h1></body></html>" 
         let parsedRequest = parser.parseRequest(request: request)
 
         XCTAssertEqual(expectedResponse, response.buildResponse(request: parsedRequest))
     }
 
+    func testBuildResponseWillReturnPOSTResponse() {
+        let request = "POST /form\r\nConnection: Keep-Alive\r\n\r\n\"My\"=\"Data\""
```

Comment:
You really need to take some time as part of your feature work to refactor how you create requests and responses in your tests. We've talked about helper methods before. You should also be able to leverage your `Request` and `Response` classes. Let me know if you need advice on doing so.